daemon/network.go

```diff
@@ -357,8 +357,8 @@ func (daemon *Daemon) DisconnectContainerFromNetwork(containerName string, netwo
 // GetNetworkDriverList returns the list of plugins drivers
 // registered for network.
 func (daemon *Daemon) GetNetworkDriverList() []string {
-	pluginList := []string{}
-	pluginMap := make(map[string]bool)
+	pluginList := []string{"overlay"}
+	pluginMap := map[string]bool{"overlay": true}
```

Comment:
perhaps, while we're at it, we could move this below the `if !daemon.NetworkControllerEnabled()` below, but let me know what you think. Also may want to wait until other people reviewed this